questions.md

```diff
@@ -0,0 +1,3 @@
+* What is a pull request?
+* What happens if a pull request has a conflict with another file someone else modified?
+* Who can approve a pull request?
```

Comment:
As an owner of a repository, you can decide who has the ability to approve pull requests. Here is some documentation on the matter: 

https://help.github.com/en/articles/about-pull-request-reviews